### PR TITLE
[TST] Add test docstrings to ml, biology, chemistry modules

### PR DESCRIPTION
**Pull request by hectormz**
# PR Description

Please describe the changes proposed in the pull request:

- Adds docstrings to tests in following modules:
    - `ml`
    - `biology`
    - `chemistry`

I did not write any of the original functions or tests, and am open to feedback for better descriptions.

**This PR partially addresses #306.**

# PR Checklist

Please ensure that you have done the following:

1. [X] PR in from a fork off your branch. Do not PR from `<your_username>`:`dev`, but rather from `<your_username>`:`<feature-branch_name>`.

2. [X] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

3. [X] Add a line to `CHANGELOG.rst` under the latest version header (i.e. the one that is "on deck") describing the contribution.
    - Do use some discretion here; if there are multiple PRs that are related, keep them in a single line.

## Quick Check

To do a very quick check that everything is correct, follow these steps below:

- [X] Run the command `make check` from pyjanitor's top-level directory. This will automatically run:
    - black formatting
    - flake8 checking
    - running the test suite
    - docs build

Once done, please check off the check-box above.

If `make check` does not work for you, you can execute the commands listed in the Makefile individually.

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

File: tests/chemistry/test_maccs_keys_fingerprint.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 )
 @pytest.mark.chemistry
 def test_maccs_keys_fingerprint(chemdf):
+    """Test conversion of SMILES strings to RDKit Mol objects of 0s and 1s."""
```
Comment (ericmjl) [inline tests/chemistry/test_maccs_keys_fingerprint.py]:
```suggestion
    """Test conversion of SMILES strings to MACCS keys fingerprints."""
```

File: tests/chemistry/test_morgan_fingerprint.py [hunk 2]
```diff
@@ -37,7 +39,8 @@ def test_morgan_fingerprint_bits(chemdf):
 )
 @pytest.mark.chemistry
 def test_morgan_fingerprint_kind_error(chemdf):
+    """Test ``morgan_fingerprint`` raises exception for invalid ``kind``."""
```
Comment (ericmjl) [inline tests/chemistry/test_morgan_fingerprint.py]: :tada:

**Comment by hectormz** [comment]
@ericmjl thanks for the feedback, just updated it!

**Comment by ericmjl** [comment]
Thanks @hectormz!

**Comment by hectormz** [comment]
More to follow...eventually...

**Comment by ericmjl** [comment]
No pressure! :smile:

**Comment by hectormz** [comment]
Bitesize. It's actually also a really nice to gain some more familiarity with parts of the codebase I'm not familiar with. So good for new and old contributors.

**Pull request merged**